targets/hasura/.gitlab-ci.yml

```diff
@@ -23,8 +23,8 @@
   variables:
     KOSKO_GENERATE_ARGS: --env dev jobs/restore
   before_script:
-    - echo "kubectl config set-context --current --namespace=${K8S_NAMESPACE}"
-    - kubectl get job.batch/restore-db-alpha || exit 0
+    - "[[ -n $FORCE ]] && kubectl -n cdtn-admin-secret delete all -l component=${CI_COMMIT_REF_SLUG} || true"
+    - kubectl -n cdtn-admin-secret get all -l component=${CI_COMMIT_REF_SLUG} && echo "Do not recreate jobs" && exit 0
```

Comment:
Why the 2nd instruction?

Reply:
@revolunet 

> Why the 2nd instruction?

- 1nd instruction : remove existing jobs config etc... from the cdtn-admin-secret on production if we `FORCE` the restoration
- 2nd instruction : Skip the job if found on the cdtn-admin-secret on production